### ISSUE TITLE
[security] Export function - change delivery from email to web

### ISSUE THREAD

**Issue by c33s** [forem/forem#1182]
follow up for #219 #576

according #576 the export is sent via email. as email communication is not encrypted by default, the export should not be sent via email.

of course the posts are public and should not contain secret information but as an export can contain unpublished posts which can contain uncleaned code including secrets, the export should not be sent via email.
instead it would be good to have a https download link sent via email and a section on the profile/dashboard/whatever page where it is possible to download the export (the download link should only be accessible if logged in).

also see comments: https://github.com/thepracticaldev/dev.to/pull/576#issuecomment-441075915

**Comment by Zhao-Andy** [comment]
Thanks, these are good points. We'll take them into consideration and think of a good solution.

**Comment by Zhao-Andy** [comment]
not stale

**Comment by cmgorton** [comment]
Hey all! Moving forward, we would love for people to head over to [forem.dev](https://github.com/forem/forem#community) for many of these feature requests that we need more discussion around. 

Since this one has been around for a while without much traction I am going to close it and encourage anyone who would like to see this feature in the future to open a discussion in [forem.dev](https://forem.dev/). There we can get more eyes on potential features and highlight any we really would love the community to contribute too. Thanks!

**Comment by c33s** [comment]
@cmgorton why not use github discussions?

**Comment by cmgorton** [comment]
Good question @c33s ! 
I proposed using Github discussions to the team internally a few weeks ago. Because we now have a community of both developers and those who are less technical (Non developers using Forem) we felt it would be better for the whole community to use a platform for communication/discussions that was less intimidating to non developers and inviting for everyone. 

There were other factors to why we chose to go this way and I'll be writing up a post soon to give more insight in to all of that. 😄
